src/website/app/pages/Theming/theming.md

```diff
@@ -1,63 +1,145 @@
+
+const MyButton = createThemedComponent(Button, {
+  Button_backgroundColor: 'tomato'
+});
 ```
 
-The themes distributed as part of Mineral UI include a value for
-`color_primary` but do not include a value for `Button_color`. In
-our component code, we leave the component-level variable
-`Button_color` as a hook for you to define if you’d like.
-Component-level theme variables start with the capitalized component
-name to differentiate from the global variables. When you’d like to
-override the Mineral UI theme at a component level in your app, you can
-use [createThemedComponent](/components/utils#create-themed-component):
+### Create your own theme
+
+Use [createTheme](#theming-api) in order to create a custom theme.  Once created, this theme can be applied using a ThemeProvider.
 
 ```jsx
+import React from 'react';
+import { render } from 'react-dom';
+import Button from 'mineral-ui/Button';
+import { createTheme, ThemeProvider } from 'mineral-ui/themes';
+
+const myTheme = createTheme('dusk');
+
+function App() {
+  return (
+    <ThemeProvider theme={myTheme}>
+      <Button>
+        Hello World
+      </Button>
+    </ThemeProvider>
+  );
+}
+
+render(<App />, document.getElementById('app'));
+```
+
+
+## API
+
+### `<ThemeProvider theme />`
+
+This component takes a theme property and provides it to the tree of components contained within.  When nested, child themes will be shallowly merged with the parent theme.
+
+See the previous examples and the [ThemeProvider](/components/theme-provider) page for more details.
+
+### `createThemedComponent(component, theme)`
+
+This function is useful when you need to override component-level theme variables.
+It is effectively the same as wrapping a ThemeProvider around a single component.
+
+**Parameters**
+
+* `component`: A React component
+* `theme`: A shallow object of theme variables or a function that accepts props and context and returns an object of theme variables
+
+**Returns**
+
+* The original React component wrapped in a ThemeProvider with the merged theme provided.
+
+**Example**
+
+```jsx
+import { createThemedComponent } from 'mineral-ui/themes';
+
 const MyButton = createThemedComponent(Button, {
-  Button_color: 'tomato'
+  Button_backgroundColor: 'tomato'
 });
 ```
 
-## Theme Variables
+### `createTheme(baseColor, overrides)`
+
+This function is useful when you want to create a new theme that uses a different color scheme or otherwise overrides default values.
+
+**Parameters**
+
+* `baseColor`: Optional.  Default: 'blue'.  Color used to generate theme color scheme.  Value must be a valid [Mineral UI color](/color/#color-ramps).
+* `overrides`: Optional.  A shallow object of variables to be spread on to the default theme.  Useful to override default values.
+
+**Returns**
+
+* A new theme object
+
+**Example**
+
+```jsx
+import { createTheme } from 'mineral-ui/themes';
+
+const myTheme = createTheme('dusk', {
+  fontFamily: 'Comic Sans MS'
```

Comment:
😝

Reply:
LOL, happy to change this to a better override if you have a suggestion. 😄

Reply:
Papyrus

Reply:
Also 😝